<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parserWidget</class>
 <widget class="QDialog" name="parserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Structure</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTreeWidget" name="boxTree">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <column>
      <property name="text">
       <string notr="true">FourCC</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTreeWidget" name="propertyTree">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <column>
      <property name="text">
       <string>Property</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="expandBoxes">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>Expand All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="collapseBoxes">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>Collapse All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="expandProperties">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>Expand All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="collapseProperties">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>Collapse All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>parserWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
